data/spec_adopters.yml

```diff
@@ -130,6 +135,15 @@ adopters:
       logo: soujava.png
       logo_white: soujava.png
       
+      
+      name: Jakarta Concurrency
```

Comment:
The preview fails due to incorrect YAML for this section. Please look at the other examples, or the template at the bottom of the file to get it correct.